src/commands/interactive/sortTenantData.ts

```diff
@@ -0,0 +1,52 @@
+import { MeshTenant } from "../../mesh/mesh-tenant.model.ts";
+import { CmdGlobalOptions } from "../cmd-options.ts";
+
+export function sortTenantDataByCost(
+  _options: CmdGlobalOptions,
+  data: MeshTenant[],
+) {
+  data.sort(function (a: MeshTenant, b: MeshTenant) {
+    let costA = 0;
+    let costB = 0;
+
+    if (a != undefined) {
+      for (const cost of a.costs) {
+        if (cost.cost != undefined && cost.cost != "") {
+          costA += Number(cost.cost);
+        }
+      }
+    }
+    if (b != undefined) {
+      for (const cost of b.costs) {
+        if (cost.cost != undefined && cost.cost != "") {
+          costB += Number(cost.cost);
+        }
+      }
+    }
+
+    if (costA > costB) return -1;
```

Comment:
looks nice, but can be even shorter! :D The sort function is defined as < 0 if less or > 0 if bigger. 0 if equals. It is actually not said that you require to return -1 or 1 exactly (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort).
So you can easily just do a:
`return costB - costA;` and are done.